src/main/resources/twilio-csharp/resource/CreateResource.mustache

```diff
@@ -6,7 +6,11 @@
                 HttpMethod.Post,
                 Rest.Domain.{{domainPackage}},
                 path,
-                postParams: options.GetParams(),
+                {{^bodyParam}}postParams: options.GetParams(),{{/bodyParam}}
+                {{#bodyParam}}
+                contentType: EnumConstants.ContentTypeEnum.JSON,
+                body: options.GetBody(),
+                {{/bodyParam}}
```

Comment:
in file [CreateOptions.mustache](https://github.com/twilio/twilio-oai-generator/pull/524/files#diff-11f9c546c0f89c61ac7c882ee15d1aaa50f7f300813f2306e9c7b8dbb13db153)
We are compariing using vendorExtension if request body has json or not. Here we are comparing with different variable.
Lets use only one way to compare.

Reply:
Is this [change](https://github.com/twilio/twilio-oai-generator/pull/524/commits/0eac16ba06fa2306b5c73bd0479896698954389d#diff-1ef958c6c48e2243826be117ed76fcf17a97a4127f7429236068ea6d7947fb88R9) correct?

Reply:
LGTM